src/extension.ts

```diff
@@ -48,7 +48,8 @@ export function activate(context: vscode.ExtensionContext) {
         const terminal = findOrCreateTerminal();
 
         terminal.show();
-        terminal.sendText(`test_launcher ${launcher_args}`);
+        const launcherOptions = vscode.workspace.getConfiguration(undefined, null).get('rubyTestLauncher.disableSpring') ? '--disable-spring' : ''
```

Comment:
From the docs, it looks like the second argument to `getConfiguration` defaults to null, so we can probably omit it.